experimental/bindings/java/com/google/iree/native/context_wrapper.cc

```diff
@@ -25,30 +25,30 @@ namespace java {
 
 namespace {
 
-std::vector<iree_vm_module_t*> GetModulesFromModuleWrappers(
-    const std::vector<ModuleWrapper*>& module_wrappers) {
-  std::vector<iree_vm_module_t*> modules(module_wrappers.size());
+std::vector<iree_vm_module_t *> GetModulesFromModuleWrappers(
```

Comment:
argh, overeager clang-format :P 

we don't want this style of spacing with `&` and `*`. We use it in compiler code (LLVM style) but not runtime code (Google style).

Do you have the fallback style set to `Google`? Or can you set it to `Google` in https://github.com/google/iree/blob/main/experimental/bindings/java/.clang-format?

In VSCode: `"clang-format.fallbackStyle": "Google",`

Reply:
Should be all good now.